CHANGELOG.asciidoc

```diff
@@ -3,6 +3,16 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.15.2]]
+=== Beats version 8.15.2
+https://github.com/elastic/beats/compare/v8.15.0\...v8.15.2[View commits]
+
+==== Breaking changes
+
+*Metricbeat*
+
+- Add GCP organization and project details to ECS cloud fields. {pull}40461[40461]
```

Comment:
Addittion of this to the changelog.next.asciidoc should be sufficient. I think that is picked up in changelog.asciidoc when the release happens. Please cross check on this.